src/event_loop.rs

```diff
@@ -185,6 +185,28 @@ impl<T> EventLoop<T> {
             event_loop_proxy: self.event_loop.create_proxy(),
         }
     }
+
+    /// Change [`DeviceEvent`] filter mode.
+    ///
+    /// Since the [`DeviceEvent`] capture can lead to high CPU usage for unfocused windows, winit
+    /// will ignore them by default on Linux/BSD. This method allows changing this filter at
+    /// runtime to explicitly capture them again.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Wayland / Windows / macOS / iOS / Android / Web**: Unsupported.
```

Comment:
I think `set_filter_device_events` could make sense on other platforms as well, so maybe `s/Unsupported/Not implemented`?

Reply:
As far as I understand, `Unsupported` means `Unsupported` by Winit. I've considered they way I should phrase it and chose to go for what most of the other Winit documentation uses. I think especially with documentation consistency matters more here and I don't want to change it everywhere.

Reply:
You're right, the rest of the docs aren't consistent on this point.